rotkehlchen/history/events.py

```diff
@@ -25,8 +25,6 @@
 log = RotkehlchenLogsAdapter(logger)
 
 
-# eth transactions, external trades, ledger actions, uniswap trades, makerDAO DSR,
-# makerDAO vaults, yearn vaults, compound, adex staking, aave lending
```

Comment:
Ai ai ai I just remembered what this comment was for. Bring it back. It's what adds up to the `HISTORY_QUERY_STEPS` number right below it.

Also please specify that the comment is for calculation of history query steps. And ... that also means you need to bump it to 11.

Perhaps need a smarter way to do that too ...